.github/workflows/zowe-cli-plugin.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]

```

Comment:
now that we have removed node 14 from the matrix, I believe we should remove the node14 specific steps in this workfow.
Here are some examples:
```yaml
    - name: Update NPM
      if: ${{ matrix.node-version == '14.x' }}
      run: npm install -g npm@7
    - name: Update Python
      if: ${{ matrix.node-version == '14.x' && matrix.os == 'macos-latest' }}
      uses: actions/setup-python@v4
      with:
        python-version: '3.10'
```